fastLatency.py

```diff
@@ -17,16 +17,14 @@
                   help="VThreshold1 DAC value for all VFATs", metavar="vt1", default=100)
 
 parser.set_defaults(nevts=1000)
-
 (options, args) = parser.parse_args()
-uhal.setLogLevelTo( uhal.LogLevel.WARNING )
 
 if options.MSPL not in range(1,9):
     print "Invalid MSPL specified: %d, must be in range [1,8]"%(options.MSPL)
     exit(1)
 
 if options.debug:
-    uhal.setLogLevelTo( uhal.LogLevel.INFO )
+    uhal.setLogLevelTo( uhal.LogLevel.DEBUG )
```

Comment:
there is pretty much no case where when debugging a tool i'm interested in seeing the `uhal::LogLevel::DEBUG` messages, `uhal::LogLevel::INFO` should be sufficient

Reply:
kk, I'll change them all to `INFO`